### PR TITLE
Sliding nav bar items

### PR DESCRIPTION
**Pull request by emilyzhong**
## Sliding Nav Bar Items
Matching sfai.org 's navbars on hover!

### Related PRs
None 

### Migrations
None

### Tests Performed, Edge Cases
None

### Screenshots
o

CC: @by-co

### CODE REVIEW

**Comment by emilyzhong** [comment]
Included dropdown menus sliding down! 
o

**Comment by emilyzhong** [comment]
sorry did not mean to close it

File: app/assets/stylesheets/components/_header.scss [hunk 1]
```diff
@@ -64,12 +60,28 @@
...
 			border-right: 0px;
 		}
 
+		&:after {
+			content: "";
+		    position: absolute;
+		    width: 100%;
+		    height: $border-stripe-height;
+		    top: 0;
+		    transition: height 0.25s ease-in-out;
```
Comment (by-co) [inline app/assets/stylesheets/components/_header.scss]: Animation nit: our header is `64px` and the SFAI's website has a header of `72px`, so we might want the transition to be proportionally faster to feel snappy — maybe `0.2s`? LOL I'M REALLY NITPICKING
